public/components/agents/stats/agent-stats.tsx

```diff
@@ -107,15 +107,15 @@ export const MainAgentStats = compose(
 
 function AgentStats({agent}){
   const [loading, setLoading] = useState();
-  const [dataStatLogcollector, setDataStatLogcollector] = useState({});
+  const [dataStatwmodulesd, setDataStatwmodulesd] = useState({});
   const [dataStatAgent, setDataStatAgent] = useState();
   useEffect(() => {
     (async function(){
       setLoading(true);
       try{
-        const responseDataStatLogcollector = await WzRequest.apiReq('GET', `/agents/${agent.id}/stats/logcollector`, {});
+        const responseDataStatwmodulesd = await WzRequest.apiReq('GET', `/agents/${agent.id}/stats/wmodulesd`, {});
```

Comment:
the stats endpoints don't change, please rollback the changes for this module

Reply:
I changed it, thank you!